src/main.rs

```diff
@@ -1,11 +1,17 @@
-use twitch_irc::login::StaticLoginCredentials;
+
+use std::fs;
+use std::str;
```

Comment:
this doesn't seem to be alphabetically ordered. have you run cargo fmt? (or rustfmt)

there is probably a way to set up a check on git hooks so you get a warning before sending the PR, I'll investigate.

Reply:
Corrected

Reply:
please rebase on top of main. we've merged the rustfmt change separately.